_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,215 @@
+
+## How will it happen? Another Call for Contributors!
+
+Old Issues:
+
+- [Call for Contributors!](https://github.com/babel/babel/issues/1347)
+- [Being Prepared for Getting Hit by a Bus](https://github.com/babel/babel/issues/1888)
+
+You might think that as [a project get more widely used](https://npm-stat.com/charts.html?package=babel-core) that more people show up to help out, but like most OSS projects that aren't backed by a company, there is a constant issue with maintenance and sustainability; people get burned out, move on to other cool projects, or get busy with work, life, family, etc.
+
+> TODO
+
+### Documentation/Website
+
+Docs contributions have definitely been lacking in the [last year](https://github.com/babel/babel.github.io/graphs/contributors?from=2015-11-29&to=2016-11-29&type=c).
+
+However just recently we've done some awesome stuff:
+
+- [@Daniel15](https://github.com/Daniel15) has been maintaining [babel-standalone](https://github.com/babel/babel-standalone) which we use in the REPL and has some automation when new releases get out.
+- [@maxiloc](https://github.com/maxiloc) added search functionality via Algolia via [#977](https://github.com/babel/babel.github.io/pull/977)
+
+and added some new collaborators: 
+
+- [@STRML](https://github.com/STRML): Added discourse to all github pages via [#875](https://github.com/babel/babel.github.io/pull/875)
+- [@xtuc](https://github.com/xtuc): Added support for reading the README from the babel repo so we don't have to sync 2 copies of docs via [#990](https://github.com/babel/babel.github.io/pull/990)
+- [@fredericmarx](https://github.com/fredericmarx): Added support for a copy to clipboard button for code snippets via [#998](https://github.com/babel/babel.github.io/pull/998)
+- [@seedofjoy](https://github.com/seedofjoy): Added a resize ability for the REPL via [#1003](https://github.com/babel/babel.github.io/pull/1003)
+
+There's so much more [we can do though](https://github.com/babel/babel.github.io/issues)!
+
+- [Reorganizing the docs layout](https://github.com/babel/babel.github.io/issues/930)
+- [Adding a resources page](https://github.com/babel/babel.github.io/issues/27)
+- [REPL: adding example dropdowns](https://github.com/babel/babel.github.io/issues/59)
+- [REPL: supporting all options](https://github.com/babel/babel.github.io/issues/92)
+- [Learn ES2015: teach others about ES2015+](https://github.com/babel/babel.github.io/issues/994)
+- [Babili: render the docs for babel-minify](https://github.com/babel/babel.github.io/issues/999)
+- [Showcase community plugins](https://github.com/babel/babel.github.io/issues/831)
+- [REPL: allow the REPL to import any community plugin from npm](https://github.com/babel/babel.github.io/issues/858)
+- [REPL: Use ASTexplorer](https://github.com/fkling/astexplorer/issues/70)
```

Comment:
👍 this is my favorite part of the post. This says a lot because I liked the post a lot too!